app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function should first check if the input string contains exactly three elements and if the first element is 'cp', as per the checklist requirement: 'Make sure you check that the input string contains 3 elements separated by space, and has `cp` on a first place.'